fboss/platform/configs/minipack3n/sensor_service.json

```diff
@@ -631,6 +715,34 @@
             "maxAlarmVal": 180
           },
           "compute": "(1000/1780)*@/1000000"
+        },
+        {
+          "name": "FAN8_TEMP",
+          "sysfsPath": "/run/devmap/sensors/MCB_VOLTAGE_MONITOR8/temp1_input",
+          "type": 3,
+          "thresholds": {
+            "upperCriticalVal": 125,
+            "maxAlarmVal": 120,
+            "minAlarmVal": -35,
+            "lowerCriticalVal": -40
+          },
+          "compute": "@/1000"
+        }
+      ]
+    },
+    {
+      "slotPath": "/SCM_SLOT@0/RUNBMC_SLOT@0",
+      "pmUnitName": "BMC",
+      "sensors": [
+        {
+          "name": "RUNBMC_THERMAL_SENSOR",
+          "sysfsPath": "/run/devmap/sensors/RUNBMC_THERMAL_SENSOR/temp1_input",
+          "type": 3,
+          "thresholds": {
+             "upperCriticalVal": 65,
+             "maxAlarmVal": 60
+           },
```

Comment:
The json linter is failing, because line:742 and line:743 have 3 spaces (instead of 2). Please fix.  `jq` might do it

Reply:
Thanks for pointing it out!
I've fixed the JSON indentation on line 742 and 743 using jq.
Please help review again, thanks!